modules/redisearch.ts

```diff
@@ -383,12 +398,14 @@ export class Redisearch extends Module {
      * @param options The additional optional parameters
      * @returns The current size of the suggestion dictionary
      */
-    async sugadd(key: string, suggestion: string, score: number, options?: FTSugAddParameters): Promise<number>{
+    async sugadd(key: string, suggestion: string, score: number, options?: FTSugAddParameters): Promise<number> {
         let args = [key, suggestion, score];
-        if(options !== undefined && options.incr !== undefined)
-            args.push('INCR');
-        if(options !== undefined && options.payload !== undefined)
-            args = args.concat(['PAYLOAD', options.payload]);
+        if (options !== undefined) {
+            if (options.incr === true)
+                args.push('INCR');
+            if (options.payload !== undefined)
+                args = args.concat(['PAYLOAD', options.payload]);
+        }
```

Comment:
Why was it bad the way it was before?

Reply:
It wasn't bad, but all of the other functions use this structure, I changed it for consistency.